README.md

```diff
@@ -76,6 +76,19 @@ When you are ready to delete the Vagrant VM use this:
 host:ae-gpcam user$ vagrant destroy
 ```
 
+If when starting up for the first time you get errors implying you can
```

Comment:
Would you like to install these packages every time the VM is built?

Reply:
I was seeing these issues with the guest utils not being available before the vagrant file had a chance try and install anything.

I'm very confused by what was going on here (as the missing packages are _inside_ of the guest not on the host).  I assume that this is something to do with getting virtualbox from arch?